googletest/src/gtest-test-part.cc

```diff
@@ -31,7 +31,7 @@
 // The Google C++ Testing and Mocking Framework (Google Test)
 
 #include "gtest/gtest-test-part.h"
-#include "src/gtest-internal-inl.h"
+#include "gtest-internal-inl.h"
```

Comment:
How is this related to the bazel change? Please revert and just leave the bazel related changes.